calculate.py

```diff
@@ -4,7 +4,7 @@
 import functions.output_display as display
 
 
-def evaluate(expression: str, is_rad: bool) -> str:
+def evaluate(expression: str, is_rad: bool, is_bool: bool) -> str:

```

Comment:
This parameter needs to be added to the docstring.

Reply:
Thanks I added it